src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -1496,4 +1497,19 @@ public TopicChange publishTopic(TopicChange topicChange) {
   
   
 
+  @Override
+  public String translate(String text) {
+    Response response = getResponse(text);
+    return response.msg;
+  }
+
+  @Override
+  public String translate(String text, String fromLang, String toLang) {
+    // FIXME implement - same mapping that current inmoov has ? en-US etc ..

```

Comment:
This comes from a more general translation interface, and I think having localization strings for translation is beneficial since different regions have different dialects. Like US English and British English

Reply:
Sure, "if supported" in the implementation .. since its free form string input nothing prevents it at this level